index.html

```diff
@@ -149,5 +149,16 @@ <h3>Skills</h3>
       </li>
     </ul>
   </section>
+  <footer class="app-contact container-v">
+    <h2>Contact me</h2>
+    <p>If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it </p>
+    <form action="https://formspree.io/f/xaykokzn" method="post" class="container-v">
+        <input class="simple-input" type="text" name="Name" maxlength="30" placeholder="Name" required>
+        <input class="simple-input" type="email" name="Email" placeholder="Email" required>
+        <textarea class="simple-input" name="Message" cols="50" rows="10" placeholder="Write your message here" required></textarea>
```

Comment:
- [ ] As stated in the project requirement , Kindly make sure that Message field accepts up to 500 chars. Please add `maxlength="500"` at the message text-area.
![Capture](https://user-images.githubusercontent.com/93598189/192618921-5093dbeb-3ad5-4e26-a919-48bfc9293cbc.PNG)